benchmarks/generate_data_footer_benchmark.py

```
@@ -0,0 +1,570 @@
+
+    Returns:
+    - A dictionary with encoding names as keys and synthetic data as values.
+    """
+    result = {}
+
+    for n_cols in [1]:
+        for n_rows in [1024 * 1, 1024 * 32, 1024 * 64]:
+            result.update(
+                {
+                    f"dec_uncompressed_opr_uint8_{n_cols}_{n_rows}": gen_dec_uncompressed_opr_uint8(
+                        n_rows, n_cols
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)